examples/previous-next-navigation/static.config.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios'
+
+export default {
+  getSiteData: () => ({
+    title: 'React Static',
+  }),
+  getRoutes: async () => {
+    const { data: posts } = await axios.get('https://jsonplaceholder.typicode.com/posts')
+    return [
+      {
+        path: '/',
+        component: 'src/containers/Home',
+      },
+      {
+        path: '/about',
+        component: 'src/containers/About',
+      },
```

Comment:
Same here. The About.js component could be moved to `pages/about.js` and the route will be inferred.